riff-raff/app/conf/context.scala

```diff
@@ -190,6 +195,13 @@ class Configuration(val application: String, val webappConfDirectory: String = "
     val token = configuration.getStringProperty("mixpanel.token")
   }
 
+  object deprecation {
+    def pauseSeconds:Option[Int] = {
+      val days = new Period(new DateTime(2017,5,22,0,0,0), new DateTime()).toStandardDays.getDays
+      if (days > 0) Some(days) else None
```

Comment:
I would be tempted to cap this at say 30 seconds. That seems irritating enough to cause an itch without causing screen smashing in an emergency.